manifests/z/Zheguisoft/IpchatGuiPc/1.0.0/Zheguisoft.IpchatGuiPc.yaml

```diff
@@ -0,0 +1,14 @@
+PackageIdentifier:  Zheguisoft.IpchatGuiPc
+PackageVersion:     1.0.0
+PackageLocale:      zh-CN
+Publisher:          Zheguisoft
+PackageName:        ipchat gui pc
+License:            Zheguisoft
+ShortDescription:   This is a chat tool
+Installers: 
+ - Architecture:    x64
+   InstallerType:   exe
+   InstallerUrl:    https://github.com/Zheguisoft/ipchat/ipchat_gui_pc.exe
+   InstallerSha256: 0bcc40e480184a22d707bbaab7bbbac7ba885676b95638cc9799ca6e18ccf729
+ManifestType:       singleton

```

Comment:
Singleton type manifests are not preferred; I suggest use tools like [wingetcreate](/microsoft/winget-create) or [YamlCreate](/microsoft/winget-pkgs/tree/master/doc/tools/YamlCreate.md) to create the manifest